static/app/views/performance/newTraceDetails/index.tsx

```diff
@@ -306,12 +318,19 @@ function TraceViewContent(props: TraceViewContentProps) {
       return loadingTrace;
     }
 
-    if (props.trace) {
-      return TraceTree.FromTrace(props.trace);
+    if (props.trace && props.status === 'success') {
```

Comment:
Why do we need to check for status here?

Reply:
I'll remove this, it was part of a debugging effort.